@xen-orchestra/xapi/src/vm.js

```diff
@@ -269,13 +271,17 @@ module.exports = class Vm {
 
   async destroy(vmRef, { deleteDisks = true, force = false, forceDeleteDefaultTemplate = false } = {}) {
     const vm = await this.getRecord('VM', vmRef)
-
     if (!force && 'destroy' in vm.blocked_operations) {
       throw new Error('destroy is blocked')
     }
 
-    if (!forceDeleteDefaultTemplate && vm.other_config.default_template === 'true') {
-      throw new Error('VM is default template')
+    if (!forceDeleteDefaultTemplate && isDefaultTemplate(vm)) {
+      throw incorrectState({
+        actual: true,
+        expected: false,
+        object: vm.$id,
+        property: 'isDefaultTemplate',
+      })
```

Comment:
Is it enough to update `other_config` or do we need to update `is_default_template` as well?

Reply:
`isDefaultTemplate` are looking of the both property `other_config` and `is_default_template`. So for me the update 
 of `other_config` is enough.
 
Btw i think it's a good idea to update `is_default_template` in another PR for synchronize these two property

Reply:
I'm talking about `isDefaultTemplate`, I'm talking about `VM_destroy`, can we *not* update `is_default_template`?

Reply:
Currently it's working good without updating `is_default template`